app/src/pages/admin/allUsersPage/usersToolbar/usersEntities.jsx

```diff
@@ -181,6 +186,9 @@ export class UsersEntities extends Component {
         removable: false,
         customProps: {
           placeholder: null,
+          inputprops: {
```

Comment:
Why not `maxlength: 256,`?

Reply:
I did it similar to other fields (line 219)